Marlin/src/pins/hc32f4/pins_AQUILA_101.h

```diff
@@ -22,7 +22,7 @@
 #pragma once
 
 //
-// Voxelab Aquila V1.0.1 and V1.0.2 (HC32F460) as found in the Voxelab Aquila X2
+// Voxelab Aquila V1.0.1/V1.0.2/V1.0.3 (HC32F460) as found in the Voxelab Aquila X2 and C2
```

Comment:
If there are links to this board's specs or schematics online, this is the best place to include those.

Reply:
hm.. well, best I can give you is what I have found. 


<p><a target="_blank" rel="noopener noreferrer nofollow"><img align="center" src="https://preview.redd.it/262qxpgw1y961.jpg?width=960&crop=smart&auto=webp&s=6c308c2180093eacd7ebbf908759f88b6b53a251" width="480" height="360" style="max-width: 100%;"></a></p>
[an early official manual picture of the board]  

notice the chip is STM32 _RE_, the chip shortage came, they went right into GD32, and so on.

<br>
<br>
<p><a target="_blank" rel="noopener noreferrer nofollow"><img align="center" src="https://user-images.githubusercontent.com/6733752/182582428-287ae7da-9bd1-4079-b4c3-f399be06a114.png" width="480" height="360" style="max-width: 100%;"></a></p>
[this is an HC32 on an Aquila C2]  

I believe the **C2** only uses the V1.0.0 - HC32

<br>
<br>
<p><a target="_blank" rel="noopener noreferrer nofollow"><img align="center" src="https://frankfurt.apollo.olxcdn.com/v1/files/zpaiahaj1dl12-RO/image;s=1000x700" width="500" height="350" style="max-width: 100%;"></a></p>
[here is V1.0.2 - HC32]  

I have a V1.0.3 - HC32 myself.

Reply:
the most common we have is GD32 and N32 chip on a V1.0.1 board.  
cant say what differences are between versions. however one clear difference is even though the first V1.0.0 with STM32 has a BL touch port like all the rest of them, except the Aquila **C2**, which uses the same HC32 but even though the board says V1.0.0 it _doesn't_ have a BL touch port.